content/posts/fpcast.md

```diff
@@ -0,0 +1,478 @@
+---
+title: "Pyodide Function Pointer Cast Removal"
+date: 2021-12-06T20:27:27-08:00
+draft: true
+tags: ["internals"]
+author: "Hood Chatham"
+# author: ["Me", "You"] # multiple authors
+showToc: true
+TocOpen: false
+draft: false
+hidemeta: false
+comments: false
+description: "Why old versions of Pyodide had a low recursion limit and how version 0.19.0 supports a much higher one."
+# canonicalURL: "https://canonical.url/to/page"
+disableHLJS: true # to disable highlightjs
+disableShare: false
+hideSummary: false
+searchHidden: true
+ShowReadingTime: true
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+cover:
+    image: "<image path/url>" # image path/url
+    alt: "<alt text>" # alt text
+    caption: "<text>" # display caption under cover
+    relative: false # when using page bundles set this to true
+    hidden: true # only hide on current single page
+---
+
+In Pyodide v0.19, we can finally support the Python default recursion limit of
+1000. We also attain a speed up between 8% to 20% for loading Pyodide and for
+individual benchmarks. The entire test suite runs faster by a comparable amount.
+The code size was reduced by 25% from 12 megabytes to 9.1 megabytes. These gains
+came from [this pull request](https://github.com/pyodide/pyodide/pull/2019)
+which removed function pointer cast emulation.
+
+Here's a graph of the recursion limit by browser and Pyodide version:
+<figure style="text-align: center;">
+  <img src="/recursion_depth.svg"  />
```

Comment:
Could you please put images under /static/posts/fpcast-removal/ subfolder and not under the root static folder, to make it easier to figure out which figure is used where.